Form-Controls/styles.css

```diff
@@ -0,0 +1,27 @@
+input {
+    padding: 5px;
+    outline: none;
+    border-radius: 5px;
+    font-size: 14px;
```

Comment:
Typically never go below 16px for text. This is because most people can't easily read small text, and we build websites for our users not for ourselves.